strongbox-event-api/src/main/java/org/carlspring/strongbox/event/artifact/ArtifactEventListenerRegistry.java

```diff
@@ -27,7 +27,7 @@ public void dispatchArtifactUploadingEvent(Path path)
         dispatchEvent(event);
     }
 
-    public void dispatchArtifactFileUpdatedEvent(Path path)
+    public void dispatchArtifactUpdatedEvent(Path path)
```

Comment:
Why do we need to change the name here? My reasoning behind this name is the fact that there may be layout providers that consider an artifact to be made up of several files. You can also have things like... metadata files being deployed, (not a good example, as we have an event for them, but I hope you get my point).

Reply:
currently this event means that whole artifact updated

> consider an artifact to be made up of several files

we can make such special event if there will be such events in the future

I thought that it's just redundant word in event name, because all other events named without `file` word.